_blog/2019/asylo-redis-sqlite.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Enclavise Redis And SQLite By Asylo"
```

Comment:
Perhaps flip the title and subtitle? (with some minor rewords and removal of quotes)

```
title: Real-World Applications in Enclaves
overview: Enclavizing Redis and SQLite
subtitle: Enclavizing Redis and SQLite
```

Reply:
Fixed in the new commit